src/test/java/world/bentobox/bentobox/listeners/flags/protection/PhysicalInteractionListenerTest.java

```diff
@@ -232,8 +238,8 @@ public void testOnPlayerInteractFarmland() {
         PlayerInteractEvent e  = new PlayerInteractEvent(player, Action.PHYSICAL, item, clickedBlock, BlockFace.UP);
         PhysicalInteractionListener i = new PhysicalInteractionListener();
         i.onPlayerInteract(e);
-        assertTrue(e.useInteractedBlock().equals(Result.DENY));
-        assertTrue(e.useItemInHand().equals(Result.DENY));
+        assertEquals(e.useInteractedBlock(), Result.DENY);
```

Comment:
This will work, but the expected result is supposed to be the first parameter.

Reply:
Oh okay, that makes sense now!

Reply:
Fixed!